R/uvhydrograph-styles.R

```diff
@@ -20,9 +28,9 @@ getUvStyle <- function(data, info, correctionLabels, plotName) {
                               points=list(x=x, y=y, pch = 21, bg = 'black', col = 'black', cex = .8, lwd=1),
                               callouts=list(x=x, y=y, labels = data$meas_Q$n, cex = .75, col='red', length = 0.05)),
 
-                appr_approved_uv = list(points = list(x=x, y=y, col="lightskyblue", type='p', pch=data[[1]]$point_type, cex=2.5, bg="lightskyblue", legend.name=legend.name, where='first')),
-                appr_inreview_uv = list(points = list(x=x, y=y, col="yellow2", type='p', pch=data[[1]]$point_type, cex=2.5, bg="yellow2", legend.name=legend.name, where='first')),
-                appr_working_uv = list(points = list(x=x, y=y, col="lightpink", type='p', pch=data[[1]]$point_type, cex=2.5, bg="lightpink", legend.name=legend.name, where='first')),
+                appr_approved_uv = list(rect = list(xleft=x0, xright=x1, ybottom=y0, ytop=y1, yheight=yheight, col="lightskyblue", bg="lightskyblue", legend.name=legend.name, where='first')),
+                appr_inreview_uv = list(rect = list(xleft=x0, xright=x1, ybottom=y0, ytop=y1, yheight=yheight, col="yellow2", legend.name=legend.name, where='first')),
+                appr_working_uv = list(rect = list(xleft=x0, xright=x1, ybottom=y0, ytop=y1, yheight=yheight, col="lightpink", bg="lightpink", legend.name=legend.name, where='first')),
```

Comment:
you shouldn't stick yheight in here - it will just cause a bunch of warnings about yheight not being a thing